Ch9.4 #1,12 Ch9.5 #2,4,5 Ch9.6 #2 Ch9.7 #2,7

In [1]:
from scipy import stats
from math import log, exp

# Chapter 9.4

**1. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with unknown mean $μ$ and known variance $1$, and it is desired to test the following hypotheses for a given number $μ_0$:**

$$H_0: μ = μ_0$$
$$H_1: μ  \ne μ_0$$

**Consider a test procedure $δ$ such that the hypothesis $H_0$ is rejected if either $\overline{X}_n ≤ c_1$ or $\overline{X}_n ≥ c_2$, and let $π(μ|δ)$ denote the power function of $δ$. Determine the values of the constants $c_1$ and $c_2$ such that $π(μ_0|δ)= 0.10$ and the function $π(μ|δ)$ is symmetric with respect to the point $μ = μ_0$.**

The distribution of the sample mean of the normal is known,

$$
\overline{X}_n \sim \text{Normal}(\mu, 1/n)
$$

(Note that I will parametrize the normal with mean and variance.)

Further, it is known that,

$$
{\overline{X}_n - \mu \over \sqrt{n}} \sim \text{Normal}(0, 1)
$$

Observe that,

$$
P(\overline{X}_n \lt c) = P\left({\overline{X}_n - \mu \over \sqrt{n}} \lt {c - \mu \over \sqrt{n}}\right)
$$

Let $\Phi$ denote the CDF of the standard normal, and let $\Phi^{-1}$ denote the corresponding inverse function. Then,

$$
P(\overline{X}_n \lt c) = \Phi\left({c - \mu \over \sqrt{n}}\right)
$$

So,

$$
P(c_1 \lt \overline{X}_n \lt c_2) = \Phi\left({c_2 - \mu \over \sqrt{n}}\right) - \Phi\left({c_1 - \mu \over \sqrt{n}}\right)
$$

**12. Let $X$ have the exponential distribution with parameter $β$. Suppose that we wish to test the hypotheses**

$$H_0: β= 1$$
$$H_1: β \ne 1$$

**We shall use a test procedure that rejects H0 if either $X ≤ c_1$ or $X ≥ c_2$.**

**a. Find the equation that must be satisfied by $c_1$ and $c_2$ in order for the test procedure to have level of significance $α_0$.**

Note that,

$$
\pi(\beta | \delta) = P(X \lt c_1 | \beta) + P(X \gt c_2 | \beta)
$$

So,

$$
\pi(\beta | \delta) = P(X \lt c_1 | \beta) + (1 - P(X \lt c_2 | \beta))
$$

Let $\Phi_\beta$ be the CDF of the exponential distribution with parameter $\beta$.

$$
\pi(\beta | \delta) = \Phi_\beta(c_1) + (1 - \Phi_\beta(c_2))
$$

Alternately, under the assumption that both $c_1$ and $c_2$ are greater than or equal to zero,

$$
\pi(\beta | \delta) = (1 - \exp(-\beta c_1)) + (1 - (1 - \exp(-\beta c_2)))
$$

$$
\pi(\beta | \delta) = 1 + \exp(-\beta c_2) - \exp(-\beta c_1)
$$

We have that

$$
\alpha_0 = \pi(\beta_0 | \delta)
$$

So,

$$
\Phi_{\beta_0}(c_1) + (1 - \Phi_{\beta_0}(c_2)) = \alpha_0
$$

Alternately,

$$
1 + \exp(-c_2) - \exp(- c_1) = \alpha_0
$$

**b. Find a pair of finite, nonzero values ($c_1$, $c_2$) such that the test procedure has level of significance $α_0 = 0.1$.**

Consider the case where $c_1 = 1/10$.Then,

$$
1 + \exp(-c_2) - \exp(-1/10) = 0.1
$$

$$
\exp(-c_2) = 0.1 - 1 + \exp(-1/10)
$$

$$
c_2 = -\ln(-0.9 + \exp(-1/10))
$$

This can be evaluated numerically.

In [2]:
c_2 = -log(-0.9 + exp(-1/10))
c_2

5.331374164240684

So, one set of values is,

$$
c_1 = 1/10
$$

$$
c_2 = -\ln(-0.9 + \exp(-1/10)) \approx 5.331374164240684
$$

# Chapter 9.5

**2. Suppose that nine observations are selected at random from the normal distribution with unknown mean $\mu$ and unknown variance $\sigma^2$, and for these nine observations it is found that $\bar X_n = 22$ and $\sum_{i=1}^n (X_i - \bar X_n)^2 = 72$**

**a.  Carry out a test of the following hypotheses at the level of significance $0.05$:**
$$
H_0: \mu \le 20,\quad
H_1: \mu > 20.
$$

Calculate the sample variance,

$$
\sigma^{'2} = {1 \over 9 - 1} \sum_{i = 1}^9 (X_i - \overline{X}_9)^2 = {72 \over 8} = 9
$$

$$
\sigma' = \sqrt{9} = 3
$$

Let,

$$
U = n^(1/2)(\bar{X}_n - \mu_0) / \sigma'
$$

Then,

$$
U = 9^(1/2)(22 - 20) / 3 = 2
$$

Under the null, $U \sim \text{T}(8)$. The p-value is given by,

$$
p = P(U \ge 2 | H_0)
$$

This can be computed numerically.

In [3]:
1 - stats.t(8).cdf(2)

np.float64(0.040258118978631297)

The p-value is 0.040258118978631297. Our significance threshold is 0.05, so we reject.

**b.  Carry out a test of the following hypotheses at the level of significance $0.05$ by using the two-sided $t$-test:**
$$
H_0: \mu = 20,\quad
H_1: \mu \neq 20.
$$


The test statistic is the same, we just double the p-value for the symmetric two-sided test.

$$
p = 2(0.040258118978631297) = 0.08051623795726259
$$

Our significance threshold is 0.05, so we not reject.


**4. Suppose that a random sample of eight observations $X_1, . . . , X_8$ is taken from the normal distribution with unknown mean $μ$ and unknown variance $σ^2$, and it is desired to test the following hypotheses:**

$$H_0: \mu = 0$$
$$H_1: \mu \ne 0$$

**Suppose also that the sample data are such that $\sum_{i = 1}^8 X_i = -11.2$ and $\sum_{i = 1}^8 X_i^2 = 43.7$. If a symmetric t test is performed at the level of significance $0.10$ so that each tail of the critical region has probability $0.05$, should the hypothesis $H_0$ be rejected or not?**

Use,

$$
\sum_{i = 1}^8 (X_i - \overline{X}_8)^2
= \sum_{i = 1}^8 X_i^2
- \frac{1}{8}\Bigl(\sum_{i = 1}^8 X_i\Bigr)^2
$$

We get,

$$
\sum_{i = 1}^8 (X_i - \overline{X}_8)^2
= 43.7
- \frac1{8}(-11.2)^2
= 28.02
$$

Find the sample standard deviation,

$$
\sigma'
= \sqrt{\frac{1}{n-1}\sum_{i = 1}^8 (X_i - \overline{X}_8)^2}
= \sqrt{\frac{1}{7}\times 28.02}
\approx 2.0015
$$

So, the test statistic is given by,

$$
U
= \frac{\overline{X}_8 - \mu_0}{\sigma'/\sqrt{8}}
= \frac{-1.4 - 0}{2.0015 / \sqrt{8}}
\approx -1.9792
$$

This is less than zero, so the two‐sided p‐value is

$$
p = 2\,P\bigl(T_{7} < -1.9792\bigr),
$$

which can be computed numerically.


In [4]:
2*stats.t(df = 7).cdf(-1.9791922591230109)

np.float64(0.08828686163751139)

We got a p-value of 0.08828686163751139. Our threshold is 0.1, so we reject.

**5. Consider again the conditions of Exercise 4, and suppose again that a t test is to be performed at the level of significance 0.10. Suppose now, however, that the t test is not to be symmetric and the hypothesis $H_0$ is to be rejected if either $U ≤ c_1$ or $U ≥ c_2$, where $Pr(U ≤ c1)= 0.01$ and $Pr(U ≥ c2)= 0.09$. For the sample data specified in Exercise 4, should $H_0$ be rejected or not?**

We have that,

$$
U \sim \text{T}(7)
$$

So if,

$$
P(U \le c_1) = 0.01
$$

Then, if we let $\Phi_7$ represent the cdf of the t-distribution with 7 degrees of freedom and $\Phi_7^{-1}$,

$$
\Phi_7(c_1) = 0.01
$$

$$
c_1 = \Phi_7^{-1}(0.01)
$$

This can be computed numerically.

In [5]:
stats.t(7).ppf(0.01)

np.float64(-2.9979515668685286)

So, our lower cut, $c_1$, is at -2.9979515668685286. The value of the test is -1.9791922591230109, which is above the cut. So, we do not reject.

# Chapter 9.6

**2. Suppose that a certain drug $A$ was administered to eight patients selected at random, and after a fixed time period, the concentration of the drug in certain body cells of each patient was measured in appropriate units. Suppose that these concentrations for the eight patients were found to be as follows:**

**1.23, 1.42, 1.41, 1.62, 1.55, 1.51, 1.60, and 1.76**

**Suppose also that a second drug $B$ was administered to six different patients selected at random, and when the concentration of drug $B$ was measured in a similar way for these six patients, the results were as follows:**

**1.76, 1.41, 1.87, 1.49, 1.67, and 1.81.**

**Assuming that all the observations have a normal distribution with a common unknown variance, test the following hypotheses at the level of significance $0.10$: The null hypothesis is that the mean concentration of drug $A$ among all patients is at least as large as the mean concentration of drug $B$. The alternative hypothesis is that the mean concentration of drug $B$ is larger than that of drug $A$.**

Observe,

$$
m = 8
$$

$$
n = 6
$$

Calculate,

$$
\overline{X}_n = {1.23 + 1.42 + 1.41+ 1.62+ 1.55+ 1.51+ 1.60+ 1.76 \over 8} = 1.5125
$$

$$
\overline{Y}_n = {1.76+ 1.41+ 1.87+ 1.49+ 1.67 + 1.81 \over 6} = 1.6683333333333332
$$

$$
S_X^2 = {1 \over 8} \sum_{i = 1}^8 (X_i - \overline{X}_n)^2 = 0.18075000000000008
$$

$$
S_Y^2 = {1 \over 6} \sum_{i = 1}^6 (Y_i - \overline{Y}_n)^2 = 0.16768333333333343
$$

We are interested in the question of whether $B$ is larger than $A$, so the test statistic is given by,

$$
U = {(m + n - 2)^{1/2}(\overline{Y}_n - \overline{X}_m) \over \left( {1 \over m} + {1 \over n} \right)^{1/2}\sqrt{S_X^2 + S_Y^2}}
$$

Calculate,

$$
U = 1.6933539163523619
$$

Under the null, $U \sim \text{T}(m + n - 2)$. We are interested in the one-sided test. So, the p-value is given by,

$$
p = P(U \ge 1.6933539163523619 | H_0)
$$

This can be computed numerically.

In [6]:
1 - stats.t(8 + 6 - 2).cdf(1.6933539163523619)

np.float64(0.05808179675220204)

The p-value is 0.05808179675220204. Our significance threshold is 0.1, so we reject.

# Chapter 9.7

**2. Suppose that a random variable $X$ has the F distribution with three and eight degrees of freedom. Determine the value of $c$ such that $Pr(X > c)= 0.975$.**

This can de determined numerically

In [7]:
stats.f(3, 8).ppf(1 - 0.975)

np.float64(0.06877632746011884)

So,

$$
P(X \gt 0.06877632746011884) \approx 0.975
$$

**7. Consider two different normal distributions for which both the means $μ_1$ and $μ_2$ and the variances $σ_1^2$ and $\sigma_2^2$ are unknown, and suppose that it is desired to test the following hypotheses:**

$$
H_0: \sigma_1^2 \le \sigma_2^2
$$

$$
H_1: \sigma_1^2 \gt \sigma_2^2
$$

**Suppose further that a random sample consisting of 16 observations for the first normal distribution yields the values $\sum_{i = 1}^{16} X_i = 84$ and $\sum_{i = 1}^{16} X_i^2 = 563$, and an independent random sample consisting of 10 observations from the second normal distribution yields the values $\sum_{i = 1}^{10} Y_i = 18$ and $\sum_{i = 1}^{10} Y_i^2 = 72$.**

**a. What are the MLE's of $\sigma_1^2$ and $\sigma_2^2$?**

We can use the MLE estimator

$$
var(X) = E[X^2] - E[X]^2
$$

$$
\hat{\sigma}_1^2 = \left({1 \over 16}\sum_{i = 1}^{16} X_i^2\right) - \left({1 \over 16}\sum_{i = 1}^{16} X_i \right)^2
$$

$$
\hat{\sigma}_1^2 = \left({563 \over 16}\right) - \left({84 \over 16}\right)^2
$$

$$
\hat{\sigma}_1^2 = 7.625
$$

$$
\hat{\sigma}_2^2 = \left({1 \over 10}\sum_{i = 1}^{10} X_i^2\right) - \left({1 \over 10}\sum_{i = 1}^{10} X_i \right)^2
$$

$$
\hat{\sigma}_2^2 = \left({72 \over 10}\right) - \left({18 \over 10}\right)^2
$$

$$
\hat{\sigma}_2^2 = 3.96
$$

**b. If an F test is carried out at the level of significance 0.05, is the hypothesis $H_0$ rejected or not?**

Find the sample variances,

$$
{S_X^2 \over m - 1} = {m \over m - 1}\hat{\sigma}_1^2 = {16 \over 15}7.625 = 8.133333333333333
$$

$$
{S_Y^2 \over n - 1} = {n \over n - 1}\hat{\sigma}_2^2 = {10 \over 9}3.96 = 4.4
$$

So, the test statistic $V$ is given by,

$$
V = {8.133333333333333 \over 4.4} = 1.8484848484848482
$$

Under the supremum of the null,

$$
V \sim F(15, 9)
$$

The p-value is given by,

$$
P(V \ge 1.8484848484848482) = 1 - P(V \lt 1.8484848484848482)
$$

This can be computed numerically.

In [8]:
1 - stats.f(15, 9).cdf(1.8484848484848482)

np.float64(0.1769969707516471)

This is not below the rejection threshold of 0.05, so we do not reject.